tools/profiler/compare-instrumentation.sh

```diff
@@ -0,0 +1,41 @@
+if [ $# -lt 2 ]; then
```

Comment:
I would recommend switching to python-based tools at some point so that you can get better logging and generally more sane error messages when things go wrong.